src/routes/expensesRoutes.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const expensesController = require('../controllers/expensesController');
+
+router.get('/expenses', expensesController.getExpenses);
+router.get('/expenses/:id', expensesController.getExpenseById);
+router.post('/expenses', expensesController.createExpense);
+router.patch('/expenses/:id', expensesController.updateExpense);
+router.delete('/expenses/:id', expensesController.deleteExpense);
```

Comment:
if you'd apply my suggestion in `createServer.js` you could omit typing `/expenses` 5 times 😄